apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/CommonUtils.cs

```diff
@@ -20,5 +21,22 @@ internal static class CommonUtils
     {
         /// <summary>A completed <see cref="Task"/>.</summary>
         internal readonly static Task CompletedTask = Task.FromResult(false);
+
+        /// <summary>The name of the this agent.</summary>
+        internal const string AgentName = "google-cloud-csharp-diagnostics";
```

Comment:
It might be better to keep the constants above all the statics, but I haven't been paying close enough attention to know what the convention is.